Managers/FrameMan.h

```diff
@@ -317,10 +317,18 @@ namespace RTE {
 		void ClearBackBuffer32() const { clear_to_color(m_BackBuffer32, 0); }
 
 		/// <summary>
-		/// Sets a specific recalculated transparency table which is used for any subsequent transparency drawing.
+		/// Sets a specific color table which is used for any subsequent blended drawing in indexed color modes.
 		/// </summary>
-		/// <param name="transSetting">The transparency setting, see the enumerator.</param>
-		void SetTransTable(TransparencyPreset transSetting);
+		/// <param name="blendMode">The blending mode that will be used in drawing.</param>
+		/// <param name="colorChannelBlendAmounts">The color channel blend amounts that will be used to select or create the correct table in the specified blending mode.</param>
+		/// <remarks>New color tables will be created using the default palette loaded at initialization because handling per-palette per-mode color tables is too much headache, even if it may possibly produce better blending results.</remarks>
```

Comment:
What don't you understand?

Reply:
The wording of the remark makes it sound like it's describing some general behaviour, rather than behaviour specific to the method. I guess it's saying that this method will create new colour tables with the default palette, but I first read it as seemingly describing the initialization process for colour tables or something.

It's kind of confusing and I just had to re-read the cpp file to remind myself what it's actually talking about, which isn't a good sign for something that's supposed to help explain things lol
But yeah:
1. I'm not sure if it's really relevant to how one uses the method, and so I'm not sure if it should be a remark in the first place.
2. More importantly, the remark is confusingly worded, and I feel like that makes the cpp file explain this concept better than the remark actually does.